services/gps2tsdb/Dockerfile

```diff
@@ -1,18 +1,34 @@
-FROM python:3
+# BUILDER
+FROM python:3 as builder
 
 WORKDIR /usr/src/app
 
 # Download latest listing of available packages:
 RUN apt-get -y update
-# Install a new package:
-RUN apt-get install -y apt-utils
-RUN apt-get install -y python3-dbus libdbus-glib-1-dev libdbus-1-dev
-RUN apt-get install -y libgirepository1.0-dev gcc libcairo2-dev pkg-config python3-dev gir1.2-gtk-3.0
-RUN ls /usr/lib/python3/dist-packages
+
+# Install packages required for DBUS dependencies
+
+# Needed for building dbus
+RUN apt-get install -y --no-install-recommends python3-dbus libdbus-glib-1-dev libdbus-1-dev
+
+# Needed for building PyGObject
+RUN apt-get install -y --no-install-recommends libgirepository1.0-dev gcc pkg-config python3-dev gir1.2-gtk-3.0
 
 COPY requirements.txt ./
-RUN pip install --no-cache-dir -r requirements.txt
+RUN pip install --user --no-cache-dir -r requirements.txt
+
+
+# RUNTIME
+FROM python:3 as runtime
```

Comment:
I think we settled on debian-slim. So maybe `python:3.8-buster-slim` or just `python:3-slim` depending on how locked down we want it to be? Or maybe I am just recalling my preference :rofl: With deploys over cell, I do think we want to be careful (but not absurd) on image size.

Interestingly it looks like `python` on dockerhub builds python from source rather than using Debian's python 3.8 https://github.com/docker-library/python/blob/44ded7c92b0c20abbeaa88785751eb0822d05ced/3.8/buster/slim/Dockerfile#L30

maybe that is to just move faster than Debian?